lib/takeaway.rb

```diff
@@ -0,0 +1,93 @@
+require './lib/menu'
+require './lib/messaging'
+
+class Takeaway
+
+attr_reader :order, :bill
+
+	def initialize
+	@menu = Menu.new
```

Comment:
menu and message could pass in as argument when initialise(dependency injection)